gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/environments/domains/ThemeResource.java

```diff
@@ -36,12 +36,7 @@
 
 import javax.validation.Valid;
 import javax.validation.constraints.NotNull;
-import javax.ws.rs.Consumes;
-import javax.ws.rs.DELETE;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
+import javax.ws.rs.*;
```

Comment:
**General comments:** Looks like intellij settings for 'class count to use import with *' is different for different team members. We should have same settings. For me and Eric I think it is a large number to avoid use '*'.

Reply:
FP